psalm-baseline.xml

```diff
@@ -53,6 +53,11 @@
       <code><![CDATA[EventInterface]]></code>
     </InvalidReturnType>
   </file>
+  <file src="src/Http/BaseApplication.php">
+    <UndefinedInterfaceMethod>
+      <code><![CDATA[events]]></code>
+    </UndefinedInterfaceMethod>
+  </file>
```

Comment:
These stan/psalm ignores are just needed, because we can't add the new method to the interface till Cake6.